tests/Transport/Smtp/PlainSmtpTransportFactoryTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+declare(strict_types=1);
+
+namespace FH\Bundle\MailerBundle\Tests\Transport\Smtp;
+
+use FH\Bundle\MailerBundle\Transport\Smtp\PlainSmtpTransportFactory;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Mailer\Transport\Dsn;
+use Symfony\Component\Mailer\Transport\Smtp\SmtpTransport;
+use Symfony\Component\Mailer\Transport\Smtp\Stream\SocketStream;
+
+class PlainSmtpTransportFactoryTest extends TestCase
```

Comment:
`final`

And extend test with:
* Test that `PlainSmtpTransportFactory` only accepts dsn's that start with `plainsmtp://`